common/ts/helpers/temp-find-method.ts

```diff
@@ -0,0 +1,380 @@
+/* istanbul ignore file */
```

Comment:
I would add a comment with a link to the GitHub issue, so that we know when to delete this file.

Reply:
Ok i added a message for that.